Source/CesiumRuntime/Private/CesiumMetadataUtilityBlueprintLibrary.cpp

```diff
@@ -59,31 +56,35 @@ UCesiumMetadataUtilityBlueprintLibrary::GetMetadataValuesForFace(
     return TMap<FString, FCesiumMetadataGenericValue>();
   }
 
-  const FCesiumMetadataModel& modelMetadata = pModel->Metadata;
-  const FCesiumMetadataPrimitive& primitiveMetadata = pGltfComponent->Metadata;
-  const TArray<FCesiumFeatureIdAttribute>& featureIdAttributes =
-      UCesiumMetadataPrimitiveBlueprintLibrary::GetFeatureIdAttributes(
-          primitiveMetadata);
+  const FCesiumPrimitiveFeatures& features = pGltfComponent->Features;
+  const TArray<FCesiumFeatureIDSet>& featureIdAttributes =
+      UCesiumPrimitiveFeaturesBlueprintLibrary::GetFeatureIDSetsOfType(
+          features,
+          ECesiumFeatureIDType::Attribute);
   if (featureIdAttributes.Num() == 0) {
     return TMap<FString, FCesiumMetadataGenericValue>();
   }
 
+  const FCesiumMetadataModel& modelMetadata = pModel->Metadata;
+  const FCesiumMetadataPrimitive& primitiveMetadata = pGltfComponent->Metadata;
+
   // For now, only considers the first feature
   // TODO: expand to arbitrary number of features once testing data is
   // available
   const TMap<FString, FCesiumFeatureTable>& featureTables =
       UCesiumMetadataModelBlueprintLibrary::GetFeatureTables(modelMetadata);
-  const FString& featureTableName =
-      UCesiumFeatureIdAttributeBlueprintLibrary::GetFeatureTableName(
-          featureIdAttributes[0]);
+  const FString& featureTableName = "";
```

Comment:
I'll fix this properly later. It'd depend on revamping `CesiumFeatureTable`, which is out of the intended scope of this PR.